apps/DemoApp/DemoApp/PlayerViewController.swift

```diff
@@ -52,6 +58,9 @@ class PlayerViewController: UIViewController {
         self.kalturaPlayer = PlayKitManager.shared.loadPlayer(pluginConfig: nil)
         self.setupKalturaPlayer()
         
+        // Setup Picture in Picture
+        self.setupPictureInPicture()
```

Comment:
Something I am considering is that this is setting up PIP for all cases, it would be nice to have test cases similar to AVPlayer so that we can isolate the PIP logic just when we want to test PIP. Otherwise it would run the basic example setup.